src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -640,7 +640,7 @@ class CollapsibleCourseRun extends React.Component {
                   />
                 }
                 extraInput={{ onInvalid: this.openCollapsible }}
-                required
+                required={jsonDeepEqual(courseSubmitInfo.targetRun, courseRun)}
```

Comment:
Thoughts on this vs `courseSubmitInfo.targetRun && courseSubmitInfo.targetRun.key === courseRun.key`?

Reply:
I think what you already have is preferred, if something else changes they wouldn't be the "same" (even though that shouldn't happen).